cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -376,6 +382,7 @@ install() {
         elif [ -n "${HOME:-}" ]; then
             _receipt_install_dir="$HOME/.cargo"
             _install_dir="$HOME/.cargo/bin"
+            _lib_install_dir="$HOME/.cargo/lib"
```

Comment:
.cargo/lib doesn't exist and i don't feel like we should be creating that...

Reply:
I'm not tied to any specific path structure, but I do think we should be keeping the lib out of the user's `PATH`. Since libraries have executable permissions, installing them in `~/.cargo/bin` means the user will get them in tab-complete suggestions (despite not being really runnable) - which feels wrong. Since `~/.cargo` is organized, we can reserve `~/.cargo/bin` for actual binaries and add a sidecar directory for libs.